src/lib.rs

```diff
@@ -71,8 +71,10 @@ use std::fmt;
 use std::num::TryFromIntError;
 use std::time::Duration;
 
-pub mod api;
+#[cfg(feature = "async")]
+use r#async::Sleeper;
```

Comment:
nit: we wouldn't need this other import if we use the full `r#async::Sleeper` below instead. Could be addressed in a future PR.